inst/stan/mvg_conj_sufficient_stats_lapse.stan

```diff
@@ -1,75 +1,78 @@
+*
+  *
+  * Modified by
+* Florian Jaeger
+* August 2020
+*/
 
 data {
-  int M;                        // number of categories
-  int L;                        // number of grouping levels (e.g. subjects)
-  int K;                        // number of features
-
-  /* Efficiency considerations: declaring n as matrix rather than array of int is supposedly faster
-     (https://mc-stan.org/docs/2_18/stan-users-guide/indexing-efficiency-section.html). It also means
-     that standard arithmetic functions can be directly applied to that matrix (unlike to more-than-1-
-     dimensional arrays). */
-  matrix[M,L] N;                // number of observations per category (m) and group (l)
-  vector[K] x_mean[M,L];        // means for each category (m) and group (l)
-  cov_matrix[K] x_ss[M,L];      // sum of uncentered squares matrix for each category (m) and group (l)
-
-  int N_test;                   // number of test trials
-  vector[K] x_test[N_test];     // locations of test trials
-  int y_test[N_test];           // group label of test trials
-  int z_test_counts[N_test,M];  // responses for test trials
-
-  int<lower=0, upper=1> m_0_known;
-  int<lower=0, upper=1> S_0_known;
-  vector[m_0_known ? K : 0] m_0_data[m_0_known ? M : 0];        // optional: user provided m_0 (prior mean of means)
-  cov_matrix[S_0_known ? K : 0] S_0_data[S_0_known ? M : 0];    // optional: user provided S_0 (prior scatter matrix of mean)
-
-  /* For now, this script assumes that the observations (cue vectors) are centered. The prior
-     mean of m_0 is set to 0. Same for the prior location parameter for the cauchy prior over
-     the variance of m_0 */
-  // separate taus for each feature to capture that features can be on separate scales:
-  // vector<lower=0>[K] tau_scales;  // scales of cauchy prior for variances along the K features (set to zero to ignore)
-  real<lower=0> tau_scale;      // scale of cauchy prior for variances of m_0 (set to zero to ignore)
-  real<lower=0> L_omega_scale;  // scale of LKJ prior for correlation of variance of m_0 (set to zero to ignore)
-}
+    int M;                        // number of categories
+    int L;                        // number of grouping levels (e.g. subjects)
+    int K;                        // number of features
+
+    /* Efficiency considerations: declaring n as matrix rather than array of int is supposedly faster
+    (https://mc-stan.org/docs/2_18/stan-users-guide/indexing-efficiency-section.html). It also means
+    that standard arithmetic functions can be directly applied to that matrix (unlike to more-than-1-
+                                                                                 dimensional arrays). */
+    matrix[M,L] N;                // number of observations per category (m) and group (l)
+    vector[K] x_mean[M,L];        // means for each category (m) and group (l)
+    cov_matrix[K] x_ss[M,L];      // sum of uncentered squares matrix for each category (m) and group (l)
```

Comment:
```suggestion
    cov_matrix[K] x_ss[M,L];      // sum of uncentered square matrices for each category (m) and group (l)
```

Reply:
Actually, it's a matrix that contains the sum of uncentered squares. So what it says was correct (for notice the "for each ..." part).